frontend/src/app/store/reducer.tsx

```diff
@@ -0,0 +1,41 @@
+
+export interface StateContext {
+    APIData: [];
+    error: string;
+    alerts: [];
+    vesrion: string;
+}
+
+const Reducer = (state, action) => {
+    switch (action.type) {
+        case 'APIData':
+            return {
+                ...state,
+                APIData: action.data
+            };
+        case 'SET_Version':
+            return {
+                ...state,
+                vesrion: action.data
```

Comment:
version?

Reply:
"dashboard version" i just added it for better understanding of global state

Reply:
oh sorry a typo

Reply:
fixed it